static/hub.ts

```diff
@@ -206,19 +207,21 @@ export class Hub {
     // Layout getters
 
     public findParentRowOrColumn(elem: GoldenLayout.ContentItem): GoldenLayout.ContentItem {
-        while (elem) {
-            if (elem.isRow || elem.isColumn) return elem;
-            elem = elem.parent;
+        let currentElem: GoldenLayout.ContentItem | null = elem;
+        while (currentElem) {
+            if (currentElem.isRow || currentElem.isColumn) return currentElem;
+            currentElem = currentElem.parent as GoldenLayout.ContentItem | null;
         }
-        return elem;
+        assert(false); // TODO
```

Comment:
I don't know what the feeling is about frontend assertions. As far as I can tell this shouldn't happen, @matsjla your insight would be appreciated. Functions which use `findParentRowOrColumn` aren't prepared to handle a null value so they would cause an error anyway.

Reply:
I'll let others decide about this, but it seems to me that we would rather send a Sentry event directly

Reply:
Sentry event sounds like a better alternative to me as well.